webv2/web.go

```diff
@@ -622,6 +623,12 @@ func migrate(w http.ResponseWriter, r *http.Request) {
 	sessionState.Conv.Audit.Progress = internal.Progress{}
 	sessionState.Conv.UI = true
 	sourceProfile, targetProfile, ioHelper, dbName, err := getSourceAndTargetProfiles(sessionState, details)
+	// TODO: Revisit the logic
+	getInfo := &utils.GetUtilInfoImpl{}
```

Comment:
we use `sessionState.GCPProjectID` at multiple places to get connection profile, set sharded config, etc and this is the value from the UI - config. I think this also needs to be changed to  `migrationProjectId`

Reply:
Changed this. I have also changed a bunch of other UI files. Please review.